controllers/rating_controller.go

```diff
@@ -110,6 +111,27 @@ func (r RatingReconciler) ratingChecker(ctx context.Context, instance *corev1alp
 	if err := r.Client.Get(ctx, types.NamespacedName{Namespace: instance.Namespace, Name: instance.Spec.ComponentName}, &component); err != nil {
 		return false, err
 	}
+
+	repository := corev1alpha1.Repository{}
+	if err := r.Client.Get(ctx, types.NamespacedName{Namespace: instance.Namespace, Name: component.Status.RepositoryRef.Name}, &repository); err != nil || !repository.Spec.EnableRating {
+		instanceDeepCopy := instance.DeepCopy()
+		cond := corev1alpha1.Condition{
+			Status:             v1.ConditionFalse,
```

Comment:
So rating's condition is only show when there are some error or rating is not enable?

Reply:
We should update condition when
1. error is not nil and  not `NotFound`
2. error is nil, but rating is disabled. 

@zqq454224016

Reply:
When the rating is disable, what error is returned?

Reply:
A simple error message is enough just like the condition message you used here.

Reply:
> So rating's condition is only show when there are some error or rating is not enable?

Actually we should track the whole running status in Rating.Let's do this in another pr